src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -0,0 +1,242 @@
+                {
+                    GUILayout.Space(9);
+                    GUILayout.Label(Styles.BigLogo, GUILayout.Height(20), GUILayout.Width(20));
+                }
+                GUILayout.EndVertical();
+
+                GUILayout.BeginVertical();
+                {
+                    GUILayout.Space(11);
+                    GUILayout.Label(Title, EditorStyles.boldLabel);
+                }
+                GUILayout.EndVertical();
+            }
+            GUILayout.EndHorizontal();
+
+            GUILayout.Space(5);
+
+            GUILayout.BeginHorizontal();
+            {
+                GUILayout.BeginVertical();
+                {
+                    GUILayout.Label("Owner");
+
+                    GUI.enabled = !isBusy;
+                    selectedOwner = EditorGUILayout.Popup(0, owners);
+                    GUI.enabled = true;
+                }
+                GUILayout.EndVertical();
+
+                GUILayout.BeginVertical(GUILayout.Width(8));
+                {
+                    GUILayout.Space(20);
+                    GUILayout.Label("/");
+                }
+                GUILayout.EndVertical();
+
+                GUILayout.BeginVertical();
+                {
+                    GUILayout.Label("Repository Name");
+                    GUI.enabled = !isBusy;
+                    repoName = EditorGUILayout.TextField(repoName);
+                    GUI.enabled = true;
+                }
+                GUILayout.EndVertical();
+            }
+            GUILayout.EndHorizontal();
+
+            GUILayout.Label("Description");
+            GUI.enabled = !isBusy;
+            repoDescription = EditorGUILayout.TextField(repoDescription);
+            GUI.enabled = true;
+            GUILayout.Space(5);
+
+            GUILayout.BeginVertical();
+            {
+                GUILayout.BeginHorizontal();
+                {
+                    GUI.enabled = !isBusy;
+                    togglePrivate = GUILayout.Toggle(togglePrivate, "Create as a private repository");
+                    GUI.enabled = true;
+                }
+                GUILayout.EndHorizontal();
+
+                GUILayout.BeginHorizontal();
+                {
+                    GUILayout.Space(5);
+                    var repoPrivacyExplanation = togglePrivate ? PrivateRepoMessage : PublicRepoMessage;
+                    GUILayout.Label(repoPrivacyExplanation, Styles.LongMessageStyle);
+                }
+                GUILayout.EndHorizontal();
+            }
+            GUILayout.EndVertical();
+
+
+            GUILayout.Space(5);
+
+            if (error != null)
+                GUILayout.Label(error, Styles.ErrorLabel);
+
+            GUILayout.FlexibleSpace();
+
+            GUILayout.BeginHorizontal();
+            {
+                GUILayout.FlexibleSpace();
+                GUI.enabled = !string.IsNullOrEmpty(repoName) && !isBusy;
+                if (GUILayout.Button("Create"))
```

Comment:
Button strings should be in `const` fields, we'll need to localize them at some point and it's easier if they're not in code.